go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/argoproj/argo
 go 1.13
 
 require (
-	bou.ke/staticfiles v0.0.0-20190225145250-827d7f6389cd
```

Comment:
this library is now deprecated and unsupported, but it won't impact us because can upgrade to Go 1.16 in Febuary which does this out of the back: https://tip.golang.org/pkg/embed/